Pipfile

```diff
@@ -6,9 +6,10 @@ verify_ssl = true
 [dev-packages]
 pytest = "*"
 tox = "*"
-flake8 = "*"
-black = "*"
+flake8 = "==3.8.4"
+black = "==20.8b1"
```

Comment:
Pin versions in Pipfile to same as the ones in .pre-commit-config.yml to avoid inconsistencies.